### PR TITLE
EARTH-214: Added editor role permissions and updated config_ignore.

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added "Editor" role and basic content permissions
- Updated config_ignore module & added ignore for `asset_injector*`

# Needed By (Date)
- Thursday

# Steps to Test

1. Checkout branch
2. composer update
3. config import
4. validate "Editor" role exists and has some content permissions
5. validate config_ignore is 2.0 version.
6. create css injector
7. config import
8. validate that css injector didnt get lost.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/default/user.role.editor.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access files overview'
+  - 'access panels in-place editing'
+  - 'access toolbar'
+  - 'administer menu'
+  - 'administer nodes'
+  - 'administer panelizer node stanford_page content'
+  - 'administer panelizer node stanford_page layout'
+  - 'bypass node access'
```
Comment (sherakama) [inline config/default/user.role.editor.yml]: Not sure about this one. Do we really want the editor to bypass node access? Can we remove this one in favor for each content type.

Comment (pookmish) [inline config/default/user.role.editor.yml]: it was kinda my quick way of checking all boxes. If we add additional node types, they will already have access to them without adding more permissions. I'm on the fence either way

Comment (sherakama) [inline config/default/user.role.editor.yml]: I think I'd rather opt-in vs opt-out over the long haul of this one.
I'll click some boxes.

**Comment by kbrownell** [comment]
Testing BLT 8.9.1 with Caravan.  Good to merge on my end.

**Pull request merged**